src/ChainSafe.Gaming/Web3/Core/Build/Web3Builder.cs

```diff
@@ -26,14 +27,17 @@ private Web3Builder()
 
             // Bind default services
             serviceCollection
-                .UseEventPoller()
+                .UseEventPoller() // todo: remove in favor of EventManager which supports WebSocket connection
```

Comment:
only for non-webgl platforms* :D

Reply:
Oh right. Then we should think of extracting IEventManager into interface and selecting the implementation that is supported on the platform at the build time.